insights/specs/insights_archive.py

```diff
@@ -121,7 +121,7 @@ class InsightsArchiveSpecs(Specs):
     ls_dev = simple_file("insights_commands/ls_-lanR_.dev")
     ls_disk = simple_file("insights_commands/ls_-lanR_.dev.disk")
     ls_edac_mc = simple_file("insights_commands/ls_-lan_.sys.devices.system.edac.mc")
-    ls_etc = simple_file("insights_commands/ls_-lan_.etc_.etc.cloud.cloud.cfg.d_.etc.nova.migration_.etc.pki.ovirt-vmconsole_.etc.pki.tls.certs_.etc.pki.tls.private_.etc.rc.d.init.d_.etc.sysconfig")
```

Comment:
To support the old archives, please use the `first_file` and add the old `simple_file` as the second alternative.

Reply:
@xiangce updated, please help to review again, thanks.